### DISCUSSION TITLE
DIYfetch - The ultimate fetch tool template utilizing Gum's join command

### DISCUSSION THREAD

**Discussion by NNBnh** [charmbracelet/gum#161] [Show and tell]
![Preview](https://user-images.githubusercontent.com/43980777/192524509-88cb892a-6851-4780-b898-467a70916473.gif)

## 💡 About

**DIYfetch** it the shell script template for writing adaptive fetch tool utilizing [Gum's join command](https://github.com/charmbracelet/gum#join).

### 📥 Download

Download the [`diyfetch`](https://github.com/info-mono/diyfetch/blob/main/diyfetch) file:

```sh
curl https://raw.githubusercontent.com/info-mono/diyfetch/main/diyfetch > ~/.local/bin/diyfetch
chmod +x ~/.local/bin/diyfetch
```

> **Note** feel free to change the file name to what ever you want.

### ⚙️ Customize

Open the fetch script with your favorite editor then add, delete and change whatever you prefer. The script contain detailed guide to customize it.

### ⌨️ Usage

Finally, run the fetch script in the terminal:

```sh
diyfetch
```

## [🖼️ Showcases](https://github.com/info-mono/diyfetch#%EF%B8%8F-showcases)

[**Check out more on the project's repository.**](https://github.com/info-mono/diyfetch)

**Comment by maaslalani** [comment]
This is really cool! Thanks so much for sharing!

**Comment by maaslalani** [comment]
You can add this to the [`examples/` folder](https://github.com/charmbracelet/gum/tree/main/examples) if you would like!

**Reply by NNBnh** [reply]
<sub>Sorry for the late reply.</sub>

Yeah sure! #163
